tests/unit/mainwindow/test_prompt.py

```diff
@@ -80,8 +80,11 @@ def test_backspacing_path(self, qtbot, tmp_path, get_prompt):
         with qtbot.wait_signal(prompt._file_model.directoryLoaded):
             for _ in range(3):
                 qtbot.keyPress(prompt._lineedit, Qt.Key_Backspace)
+        prompt._set_fileview_root(prompt._lineedit.text())
```

Comment:
I don't understand why this is needed - I can see the test is failing without it, but why is this needed now and wasn't before?

Reply:
Completely unclear. For some reason when a key is pressed by the qtbot this function isn't always run. I spent hours trying to figure out any possible reason why it isn't being run and the only one is that qtbot.keyPress doesn't trigger `textEdited`